### PR TITLE
Added an example of contentById query

### PR DESCRIPTION
**Pull request by biapar**

### CODE REVIEW

**Comment by nikcio** [comment]
Thank you for the contribution!

Please change the file name to: getContentByIdWithVariables.md and then we can merge it.

**Comment by biapar** [comment]
> Thank you for the contribution!
> 
> Please change the file name to: getContentByIdWithVariables.md and then we can merge it.

Changed

File: docs/v3/README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 ## Content queries
 
+* [GetContentById with variables](contentQueries/getContentByIdWithVariabiles.md)
```
Comment (nikcio) [inline docs/v3/README.md]: The link needs to match the file 😄

**Review by nikcio — CHANGES_REQUESTED** [review]

File: docs/v3/README.md (hunk 1 above)
Comment (biapar) [reply docs/v3/README.md]: Done

**Pull request merged**